internal/pkg/cli/deploy/workload.go

```diff
@@ -67,7 +67,9 @@ func (noopActionRecommender) RecommendedActions() []string {
 	return nil
 }
 
-type imageBuilderPusher interface {
+type RepositoryService interface {
+	URI() (string, error)
```

Comment:
it doesn't make sense to me when a function seems to only be used within the other pkg, but the interface is declared in this pkg.

Reply:
Done!